modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java

```diff
@@ -56,15 +57,15 @@ public void portalInstanceRegistered(Company company) throws Exception {
 		_addAssetVocabulary(
 			company, "audience",
 			AssetVocabularyConstants.VISIBILITY_TYPE_INTERNAL,
-			JournalArticle.class);
+			JournalArticle.class, DLFileEntry.class);
```

Comment:
Sort these alphabetically (DL... goes before Journal...).

Reply:
The associated asset types are shown in the same order as they are introduced, so to keep it as we expected (first web content and then documents) this is the way of doing it :)